baseplate/ratelimit/backends/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .base import RateLimitBackend
+
+
+__all__ = [
+    'RateLimitBackend',
+]
```

Comment:
redis & memcache backends not exported here since that would trigger an import of `redis` and `pymemcache`